diff --git a/node_modules/@ledgerhq/hw-transport/lib/Transport.js b/node_modules/@ledgerhq/hw-transport/lib/Transport.js
index 9317ebd..d22f272 100644
--- a/node_modules/@ledgerhq/hw-transport/lib/Transport.js
+++ b/node_modules/@ledgerhq/hw-transport/lib/Transport.js
@@ -46,7 +46,7 @@ class Transport {
          *    to detect unresponsive device (see `unresponsiveTimeout`). This timeout aborts the exchange.
          * @returns {Promise<Buffer>} A promise that resolves with the response data from the device.
          */
-        this.send = (cla, ins, p1, p2, data = Buffer.alloc(0), statusList = [errors_1.StatusCodes.OK], { abortTimeoutMs } = {}) => __awaiter(this, void 0, void 0, function* () {
+        this.send = (cla, ins, p1, p2, data = Buffer.alloc(0), statusList = [errors_1.StatusCodes.OK], { abortTimeoutMs } = { abortTimeoutMs: 2 * 60 * 1000 }) => __awaiter(this, void 0, void 0, function* () {
             const tracer = this.tracer.withUpdatedContext({ function: "send" });
             if (data.length >= 256) {
                 tracer.trace("data.length exceeded 256 bytes limit", { dataLength: data.length });
