diff --git a/node_modules/@polkadot/util/cjs/versionDetect.js b/node_modules/@polkadot/util/cjs/versionDetect.js
index 501f104..9173933 100644
--- a/node_modules/@polkadot/util/cjs/versionDetect.js
+++ b/node_modules/@polkadot/util/cjs/versionDetect.js
@@ -74,7 +74,7 @@ function warn(pre, all, fmt) {
  * @description A `@polkadot/*` version detection utility, checking for one occurence of a package in addition to checking for ddependency versions.
  */
 function detectPackage({ name, path, type, version }, pathOrFn, deps = []) {
-    if (!name.startsWith('@polkadot')) {
+    if (!name.startsWith('@polkadot') && !name.startsWith('@subwallet')) {
         throw new Error(`Invalid package descriptor ${name}`);
     }
     const entry = getEntry(name);
diff --git a/node_modules/@polkadot/util/versionDetect.js b/node_modules/@polkadot/util/versionDetect.js
index b76c020..a6d1bed 100644
--- a/node_modules/@polkadot/util/versionDetect.js
+++ b/node_modules/@polkadot/util/versionDetect.js
@@ -71,7 +71,7 @@ function warn(pre, all, fmt) {
  * @description A `@polkadot/*` version detection utility, checking for one occurence of a package in addition to checking for ddependency versions.
  */
 export function detectPackage({ name, path, type, version }, pathOrFn, deps = []) {
-    if (!name.startsWith('@polkadot')) {
+    if (!name.startsWith('@polkadot') && !name.startsWith('@subwallet')) {
         throw new Error(`Invalid package descriptor ${name}`);
     }
     const entry = getEntry(name);
