diff --git a/node_modules/react-tooltip/dist/index.es.js b/node_modules/react-tooltip/dist/index.es.js
index 0ce3064..affd243 100644
--- a/node_modules/react-tooltip/dist/index.es.js
+++ b/node_modules/react-tooltip/dist/index.es.js
@@ -1720,7 +1720,7 @@ function windowListener (target) {
    */
   target.prototype.onWindowResize = function () {
     if (!this.mount) return;
-    this.hideTooltip();
+    // this.hideTooltip();
   };
 }
 
