diff --git a/node_modules/react-qrcode-logo/dist/index.d.ts b/node_modules/react-qrcode-logo/dist/index.d.ts
index 9349341..5d775b1 100644
--- a/node_modules/react-qrcode-logo/dist/index.d.ts
+++ b/node_modules/react-qrcode-logo/dist/index.d.ts
@@ -30,6 +30,7 @@ export interface IProps {
     qrStyle?: 'squares' | 'dots';
     style?: object;
     id?: string;
+    ignoreEncode?: boolean;
 }
 export declare class QRCode extends React.Component<IProps, {}> {
     private canvas;
diff --git a/node_modules/react-qrcode-logo/dist/index.js b/node_modules/react-qrcode-logo/dist/index.js
index cfca6de..6052aa2 100644
--- a/node_modules/react-qrcode-logo/dist/index.js
+++ b/node_modules/react-qrcode-logo/dist/index.js
@@ -168,7 +168,12 @@ var QRCode = /** @class */ (function (_super) {
         var logoHeight = this.props.logoHeight ? +this.props.logoHeight : 0;
         var logoPadding = this.props.logoPadding ? +this.props.logoPadding : 0;
         var qrCode = qrGenerator(0, ecLevel);
-        qrCode.addData(QRCode.utf16to8(value));
+        if (!this.props.ignoreEncode) {
+          qrCode.addData(QRCode.utf16to8(value));
+        }
+        else {
+          qrCode.addData(value);
+        }
         qrCode.make();
         var canvas = ReactDOM.findDOMNode(this.canvas.current);
         var ctx = canvas.getContext('2d');
@@ -299,7 +304,8 @@ var QRCode = /** @class */ (function (_super) {
         logoOpacity: 1,
         qrStyle: 'squares',
         eyeRadius: [0, 0, 0],
-        logoPaddingStyle: 'square'
+        logoPaddingStyle: 'square',
+        ignoreEncode: false,
     };
     return QRCode;
 }(React.Component));
diff --git a/node_modules/react-qrcode-logo/lib/index.tsx b/node_modules/react-qrcode-logo/lib/index.tsx
index 0a0ee2e..51cb010 100644
--- a/node_modules/react-qrcode-logo/lib/index.tsx
+++ b/node_modules/react-qrcode-logo/lib/index.tsx
@@ -36,6 +36,7 @@ export interface IProps {
     qrStyle?: 'squares' | 'dots';
     style?: object;
     id?: string;
+    ignoreEncode?: boolean;
 }
 
 interface ICoordinates {
@@ -58,7 +59,8 @@ export class QRCode extends React.Component<IProps, {}> {
         logoOpacity: 1,
         qrStyle: 'squares',
         eyeRadius: [0, 0, 0],
-        logoPaddingStyle: 'square'
+        logoPaddingStyle: 'square',
+        ignoreEncode: false,
     };
 
     private static utf16to8(str: string): string {
@@ -260,7 +262,11 @@ export class QRCode extends React.Component<IProps, {}> {
         const logoPadding = this.props.logoPadding ? +this.props.logoPadding : 0;
 
         const qrCode = qrGenerator(0, ecLevel);
-        qrCode.addData(QRCode.utf16to8(value));
+        if (!this.props.ignoreEncode) {
+          qrCode.addData(QRCode.utf16to8(value));
+        } else {
+          qrCode.addData(value);
+        }
         qrCode.make();
 
         const canvas: HTMLCanvasElement = ReactDOM.findDOMNode(this.canvas.current) as HTMLCanvasElement;
